### DISCUSSION TITLE
Plot grain boundaries on .tiff image

### DISCUSSION THREAD

**Discussion by tongzhangqu** [mtex-toolbox/mtex#1741] [Ask Anything]
Hi, 

I am trying to plot grain boundaries from MTEX on a forescatted electron image in tiff format. This is my code:

```
figure(1)
forescatter_img = imread('XXXXXX.tiff');
forescatter_img = forescatter_img(:,:,1:3);
imshow(forescatter_img)
saveas(gcf,'forescatter_img.fig') %save tiff as fig
close(gcf);

figure(2)
openfig('forescatter_img.fig')
hold on
plot(grains.boundary, 'linewidth', 2);
text(grains,int2str(grains.id),'color','black','fontSize',25); % show grain id
hold off
```
The tiff and results from mtex are always separated. Is there a way to plot MTEX results on an existed tiff image?

Thank you
Tongzhang

**Comment by kilir** [comment]
Hi,
since imagesc/imshow plots to a normal matlab figure and mtex to an mtexFigure, you may get an error when plotting directly to an mtexFigure but e.g. this works fine
```
img = imread("kobi.png");
mtexdata titanium
f=newMtexFigure;
imagesc(img)
hold on
plot(ebsd('t'),ebsd('t').orientations,'FaceAlpha',0.6)
hold off
```
Why do you want to save it as an fig actually?
![out](https://user-images.githubusercontent.com/11697557/221274717-165f1315-ba1f-491f-8e89-956a0d929f78.png)


Cheers,
Rüdiger

**Reply by tongzhangqu** [reply]
Hi Rüdiger,

Thank you very much. I was not aware of the difference between matlab figure and MTEX figure. 

The codes work well. 

Best wishes
Tongzhang
